docs/aboutstudio/upload/upload.mdx

```diff
@@ -1,7 +1,26 @@
 ---
-title: Upload 101
+title: Upload
 ---
```

Comment:
"Livepeer Studio Uploading", or
"Uploading with Livepeer Studio"
"Livepeer Studio Uploading Feature"???
@pglowacky should probably chime in for a productization perspective